### ISSUE TITLE
Android 6.0.1 camera open failed

### ISSUE THREAD

**Issue by Ace2222** [jitsi/jitsi-meet#2209]
![image](https://user-images.githubusercontent.com/32385369/33162010-154b2ca2-d062-11e7-9ec9-6a84374941ab.png)

![image](https://user-images.githubusercontent.com/32385369/33162089-95486c12-d062-11e7-8a55-dc54d0768707.png)

**Comment by Ace2222** [comment]
Does anyone know why?

**Comment by saghul** [comment]
Does your device have 1 or 2 cameras?

**Comment by Ace2222** [comment]
1 camera

**Comment by saghul** [comment]
I suspect the problem is we try to use the front camera, but it doesn't exist and thus it fails. Does it work if you click the camera switch button? It's on the side toolbar.

**Comment by Ace2222** [comment]
No,it doesnot work

**Comment by saghul** [comment]
Thanks for confirming. It may take a while for us to solve this, I'm not sure how we can even test it...

**Comment by RealWearDavid** [comment]
I can test this on a single camera Android device.  Please PM me.  My company really wants this to work as well.  www.realwear.com

**Comment by saghul** [comment]
Hey @RealWearDavid, this is now fixed with the last commit. Kudos to @guusdk for fixing it. We tested it on a HMT-1 and it works now.

**Comment by saghul** [comment]
@Ace2222 Latest master should also fix your problem.

**Comment by RealWearDavid** [comment]
@saghul @guusdk What version release was it fixed on?  I am using 1.14.75 which seems to be the latest available on the APK depositories.  I cannot download from Google Play.  If there is an update would someone post a link to download the new APK please?  Thanks!

**Comment by saghul** [comment]
It’s not released yet, but if you build from master right now the bug should be fixed. It will be part of the next release, for which I have no ETA.

**Comment by RealWearDavid** [comment]
@saghul I'm having troubles building from master right now.  Would you or @guusdk be kind enough to attached the new APK build here?

**Comment by saghul** [comment]
@RealWearDavid sorry, I can't provide you with an apk, as I was running in debug mode with a hacked build which always entered a given room. This was necessary because the UI is not navigable because the accessibility labels are not recognized. FWIW, we changed the one for the hangup button and made the toolbar always visible and we could hangup with a voice command. Pretty neat!

**Comment by RealWearDavid** [comment]
@saghul @guusdk Anyone want to optimize and build a custom APK and name it "Remote Mentor"?  It is super easy to optimize this app for voice.  See our developer portal here: developer.realwear.com.  I tested the new release on the HMT-1 at RealWear HQ and we are very impressed.  I walked across the street and still kept connectivity.  This is huge because this matches companies working with Fortune 100 companies right now.  Since RealWear is a hardware company we would interested in bundling Jitsi on the device for free.
